sanic/request.py

```diff
@@ -7,7 +7,11 @@
 try:
     from ujson import loads as json_loads
 except ImportError:
-    from json import loads as json_loads
+    from json import loads as _json_loads
+
+    def json_loads(i):
+        i = i.decode() if isinstance(i, bytes) else i
+        return _json_loads(i)
```

Comment:
This just an FYI. But Python 3.6 added support for `json.loads()` on `bytes`. Not that it helps much since we also support 3.5.

```python
>>> import json
>>> json.loads(b'{"foo": "bar"}')
{'foo': 'bar'}
```

Anyway, this got me thinking a bit. Is the `isinstance` check really necessary? Since it's only used to parse the HTTP body, which should always be in `bytes`, right?